notebook/static/base/js/i18nload.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Plugin to load a single locale.
+ */
+define([
+	"require",
+	"module",

```

Comment:
module is part of requirejs, which allows us to define the name of the module for the plugin. Remember, a plugin has two parts, the javascript module ( in this case i18nload ), and the argument ( the stuff after the ! ). So the plugin defines its module id in the object it passes back, and then i18n.js does a require on the module.id.